src/main/java/chatterbot/parser/Parser.java

```diff
@@ -0,0 +1,105 @@
+package chatterbot.parser;
+
+import chatterbot.storage.Storage;
+import chatterbot.ui.Ui;
+import chatterbot.data.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Parser {
+
+    public static void evaluateCommand(String userMessage, Ui ui, ArrayList<Task> list, Storage storage, String file, TaskList taskList) {
```

Comment:
This method is very long -- would it make the code clearer to split it up into smaller methods? You can consider creating a Command class as outlined in the [A-MoreOOP](https://nus-cs2103-ay2324s1.github.io/website/schedule/week3/project.html#a-moreoop) extension stretch goal to separate the command parsing and execution.